package.json

```diff
@@ -63,8 +63,9 @@
     "@types/react-test-renderer": "17.0.2",
     "@types/react-timeago": "4.1.3",
     "@types/semver": "7.3.12",
+    "@uiw/codemirror-theme-duotone": "4.11.5",
     "@uiw/react-codemirror": "4.11.5",
-    "@vitejs/plugin-react": "1.3.2",
+    "@vitejs/plugin-react": "^2.0.1",
```

Comment:
Could remove the `^` to match the rest of the file.

Reply:
Thanks, addressed in https://github.com/Unleash/unleash-frontend/pull/1237/commits/0f672fc3a25630c340431b04bc3845c9d276090a.

Regarding the coordination with the in-progress frontend/backend repo merge, should I merge this PR right away?

Reply:
Might be easier to wait for the merge and then do the same change there. Should be pretty close!

Reply:
Here it is: https://github.com/Unleash/unleash/pull/1977